deployer/update_grafana_datasources.py

```diff
@@ -0,0 +1,240 @@
+"""
+### Summary
```

Comment:
Could rename this to something like `update_central_grafana_datasources.py`, or put this in a folder that makes it clear it only applies to the central grafana and not each support chart grafana?

Reply:
Done in https://github.com/2i2c-org/infrastructure/pull/1215/commits/6f0c993e444bb7f6b6dbd9890bb808ea2f341d27